apk/src/utils.rs

```diff
@@ -4,16 +4,18 @@ use anyhow::{Context, Result};
 #[repr(u8)]
 pub enum Target {
     ArmV7a = 1,
-    Arm64V8a = 2,
-    X86 = 3,
-    X86_64 = 4,
+    Armeabi = 2,
+    Arm64V8a = 3,
+    X86 = 4,
+    X86_64 = 5,
```

Comment:
Changing the `repr` values here might not be desired, but I need to check where it is even used.

Reply:
alright